Sources/OpenAPI/Generated/Entities/Platform.swift

```diff
@@ -1,13 +1,10 @@
 // Generated by Create API
 // https://github.com/CreateAPI/CreateAPI
-//
-// swiftlint:disable all
 
 import Foundation
 
 public enum Platform: String, Codable, CaseIterable {
 	case ios = "IOS"
 	case macOs = "MAC_OS"
 	case tvOs = "TV_OS"
-    case visionOs = "VISION_OS" // VisionOS manually added since version 3.0 of Apple's OpenAPI spec fails to include it. Reported in FB13208526 27/9/2023.
```

Comment:
FYI, we need to manually keep this line until Apple fixes `VISION_OS` missing in the spec or we come up with a better solution to handle this.

Reply:
Thanks, just added it.